openai/components/oai_finetune_pipeline_component.yml

```diff
@@ -0,0 +1,114 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineComponent.schema.json
+name: microsoft_com_azureml_oaie_finetune_pipeline_v2
+version: 0.0.1
+type: pipeline
+display_name: PipelineComponent for OpenAI Finetuning workflow
+description: Pipeline component for OpenAI finetuning workflow
+
+inputs:
+  model:
+    type: string
+    optional: False
+    default: Ada
+    description: GPT-3 model engine
+    enum:
+      - Ada
+      - Babbage
+      - Curie
+      - DaVinci
+      - Text-Ada-001
+      - Text-Babbage-001
+      - Text-Curie-001
+      - Text-DaVinci-001
+      - Text-DaVinci-002-onebox-inference-only
+      - Text-DaVinci-fine-tune-002
+      - Text-DaVinci-003-onebox-inference-only
+      - Text-Chat-DaVinci-002-onebox-inference-only
+      - ChatGPT-202301-onebox-inference-only
+      - ChatGPT-202301-2-8k-onebox-inference-only
+      - Code-Cushman-001
+      - Code-Cushman-fine-tune-002
+      - Codex-Cushman-fim-2021-11-10
+      - Code-DaVinci-002-onebox-inference-only
+      - Code-DaVinci-fine-tune-002
+      - Text-Similarity-Ada-001-onebox-inference-only
+      - Text-Similarity-Babbage-001-onebox-inference-only
+      - Text-Similarity-Curie-001-onebox-inference-only
+      - Text-Similarity-DaVinci-001-onebox-inference-only
+      - Code-Search-Ada-Code-001-onebox-inference-only
+      - Code-Search-Ada-Text-001-onebox-inference-only
+      - Code-Search-Babbage-Code-001-onebox-inference-only
+      - Code-Search-Babbage-Text-001-onebox-inference-only
+      - Text-Search-Ada-Doc-001-onebox-inference-only
+      - Text-Search-Ada-Query-001-onebox-inference-only
+      - Text-Search-Babbage-Doc-001-onebox-inference-only
+      - Text-Search-Babbage-Query-001-onebox-inference-only
+      - Text-Search-Curie-Doc-001-onebox-inference-only
+      - Text-Search-Curie-Query-001-onebox-inference-only
+      - Text-Search-DaVinci-Doc-001-onebox-inference-only
+      - Text-Search-DaVinci-Query-001-onebox-inference-only
+      - Text-Embedding-Ada-002-onebox-inference-only
+      - Text-Embedding-Ada-002-test-stage-onebox-inference-only
+      - Code-Ada-002-test-stage
+  train_dataset:
+    type: uri_folder
+    optional: False
+    description: Input dataset (file or folder). If a folder dataset is passed, includes all nested files.
+  registered_model_name:
+    type: string
+    optional: False
+    description: User-defined registered model name
+  singularity_cpu_instance_type:
```

Comment:
For 3P components we need serverless only.

Reply:
Updating the component for 3P